docs/pre-release/argo/canary-rollouts-with-argo.md

```diff
@@ -0,0 +1,262 @@
+# Argo Rollouts and Edge Stack Quick Start
+
+This tutorial will walk you through the process of configuring Argo Rollouts to work with Edge Stack to facilitate canary releases. This will enable users to safely [rollout new versions](https://blog.getambassador.io/deploying-argo-rollouts-for-canary-releases-on-kubernetes-f5910ed1fd61) of services on Kubernetes.
+
+## 1. Install and configure Edge Stack
+
+You'll need to first install Edge Stack in your cluster. Follow the [Edge Stack installation](/docs/latest/tutorials/getting-started) to install Edge Stack via Kubernetes YAML, Helm, or the command-line installer in your cluster.
+
+By default, Edge Stack routes via Kubernetes services. For best performance with canaries, we recommend you use endpoint routing. Enable endpoint routing on your cluster by saving the following configuration in a file called `resolver.yaml`:
+
+```
+apiVersion: getambassador.io/v2
+kind: KubernetesEndpointResolver
+metadata:
+  name: endpoint
+```
+
+Apply this configuration to your cluster: `kubectl apply -f resolver.yaml`.
+
+## 2. Install and configure Argo Rollouts
+
+If you're using Google Kubernetes Engine, grant your account the ability to create new Cluster Roles:
+
+```
+kubectl create clusterrolebinding YOURNAME-cluster-admin-binding --clusterrole=cluster-admin --user=YOUREMAIL@gmail.com
+```
+
+Now, install the Argo Rollouts controller in your cluster. Note that Argo must be installed in the `argo-rollouts` namespace.
+
```

Comment:
perhaps mention that they are installing our fork of Argo (for now) - and add a pointer to the repo of the fork?